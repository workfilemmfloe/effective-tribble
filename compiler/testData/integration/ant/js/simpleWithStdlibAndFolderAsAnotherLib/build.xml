<project name="Ant Task Test" default="build">
  <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>
  <property name="library.path" value="${test.data}/jslib-example"/>
  <target name="build">
    <kotlin2js src="${test.data}/root1" output="${temp}/out.js" main="call">
      <libraries>
        <pathelement path="${library.path}"/>
      </libraries>
    </kotlin2js>
    <copy todir="${temp}">
      <fileset dir="${library.path}">
        <include name="**/*.js"/>
        <exclude name="META-INF/**"/>
      </fileset>
    </copy>
  </target>
</project>
