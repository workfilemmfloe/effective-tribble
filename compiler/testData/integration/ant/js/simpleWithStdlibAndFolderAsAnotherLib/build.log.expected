OUT:
Buildfile: [TestData]/build.xml

build:
[kotlin2js] Compiling [[TestData]/root1] => [[Temp]/out.js]
     [copy] Copying 1 file to [Temp]

BUILD SUCCESSFUL
Total time: [time]

Return code: 0
